src/main/java/de/exercises/pithonAPI/PiThonAPI.java

```diff
@@ -0,0 +1,56 @@
+package de.exercises.pithonAPI;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class PiThonAPI {
+    public static final String PITHON_URL = "http://localhost:5000/api";
+
+    public static void main(String[] args) {
+        if(args[0] == null) {
+            System.out.println("argument(s) missing: number (pi, e, sqrt2) and amount(optional)");
```

Comment:
For future reading, you could look at standards for argument specifications.
It should probably be something like `pi|e|sqrt2 [amount]`